src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer.md

```diff
@@ -90,6 +90,8 @@ Repeat mode enabled by default. </td>
 </table>
 </div>
 
+//TODO add redaction related buttons
```

Comment:
I'm assuming the TODOs on this page are about updating the nxasset? @bchauvin do you manage it?

Reply:
Yes, thanks for reminding me! Forgot this one, done now in my latest commit.